src/ui/interfaces/ConfirmationProps.ts

```diff
@@ -2,7 +2,7 @@ export interface ConfirmationProps {
     integrationName: string
 }
 
-interface PIIdefinition {
+export interface PIIdefinition {
```

Comment:
Consider changing PII to `UserInputDefinition`, `CredentialIputDefinition` since not all possible credentials will be PII.